app/client/public/index.html

```diff
@@ -199,6 +199,19 @@
         white-space: normal;
       }
 
+      button:disabled {
+        background-color: #c9c9c9;
+        color: #454545;
+        cursor: not-allowed;
+        opacity: 1;
+      }
+
+      button:&not(:disabled):hover,
+      button:&not(:disabled):focus {
```

Comment:
This isn't having an effect currently, it should be changed to:
```suggestion
      button:not(:disabled):hover,
      button:not(:disabled):focus {
```

Reply:
Good catch!